examples/complete/variables.tf

```diff
@@ -0,0 +1,174 @@
+  description = "Prefix to append to all resources created by this example."
+  default     = "example-cis"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "The name of an existing resource group to provision resources in to. If not set a new resource group will be created using the prefix variable"
+  default     = null
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources."
+  default     = []
+}
+
+variable "plan" {
+  type        = string
+  description = "Plan for the CIS instance. standard-next or trial."
+  default     = "trial"
+}
+
+variable "domain_name" {
+  type        = string
+  description = "Domain name to be added for CIS Instance."
+  default     = "example.com"
+}
+
+variable "dns_record_set" {
+  description = "List of DNS records to be added for CIS Instance."
+  type = list(object({
+    name     = optional(string)
+    type     = string
+    ttl      = optional(number)
+    content  = optional(string)
+    priority = optional(number)
+    data = optional(object({
+      altitude       = optional(number)
+      lat_degrees    = optional(number)
+      lat_direction  = optional(string)
+      lat_minutes    = optional(number)
+      lat_seconds    = optional(number)
+      long_degrees   = optional(number)
+      long_direction = optional(string)
+      long_minutes   = optional(number)
+      long_seconds   = optional(number)
+      precision_horz = optional(number)
+      precision_vert = optional(number)
+      size           = optional(number)
+      tag            = optional(string)
+      value          = optional(string)
+      port           = optional(number)
+      priority       = optional(number)
+      proto          = optional(string)
+      service        = optional(string)
+      target         = optional(string)
+      weight         = optional(number)
+      name           = optional(string)
+    }))
+  }))
+  default = [
+    {
+      type    = "A"
+      name    = "test-example"
+      content = "1.2.3.4"
+      ttl     = 900
+    }
+  ]
+}
+
+variable "glb_name" {
+  description = "Name of CIS Global Load Balancer."
+  type        = string
+  default     = "glb"
+}
+
+variable "glb_description" {
+  description = "Description of CIS Global Load Balancer."
+  type        = string
+  default     = "Load Balancer"
+}
+
+variable "glb_enabled" {
+  description = "To enable/disable CIS Global Load Balancer. If set to true, the load balancer is enabled and can receive network traffic."
+  type        = bool
+  default     = true
+}
+
+variable "ttl" {
+  description = "Time-to-live(TTL) in seconds of CIS Global Load Balancer(GLB). Allowed value is 120 or greater if GLB is not proxied otherwise it is automatically set."
+  type        = number
+  default     = 120
+}
+
+variable "origin_pools" {
+  description = "List of origins with an associated health check to be created for CIS Global Load Balancer."
```

Comment:
please change all instances `Global Load Balancer` to lowercase. It is a generic term, not a proper noun or product. see https://cloud.ibm.com/docs/cis?topic=cis-configure-glb

Reply:
done